lib/vue/src/components/webcam/CWebcamModal.vue

```diff
@@ -0,0 +1,170 @@
+<template>
+  <b-modal
+    ref="modal"
+    size="lg"
+    centered
+    :title="modalTitle"
+    :body-class="processingWebcam ? 'd-flex flex-column align-items-center' : 'p-0'"
```

Comment:
Instead of this, just style the spinner instead. You can wrap it inside a div and make it a big as the camera is gonna be when it's loaded. (so the modal doesn't resize)